lib/tasks/create_fake_data.rake

```diff
@@ -0,0 +1,89 @@
+namespace :db do
+  namespace :seed do
+    desc 'Generate fake patient entries for data wranglers'
+    task :create_fake_data => :environment do
+
+        users = User.all
+        clinics = Clinic.all
+
+        5000.times do |idx|
+          flag = (rand(10) % 5 == 0)
+
+          initial_call = Date.today - rand(1000)
+          
+          has_appt = rand(3) % 2 == 0
+
+          has_pledge = rand(3) != 2 
+
+          lines = ['DC', 'VA', 'MD'] # need to add Spanish maybe? 
+
+          patient = Patient.create!(
+            name: 'Randomized Patient',
+            primary_phone: SecureRandom.random_number(10**10).to_s.rjust(10, '0'), 
+            initial_call_date: initial_call,
+            created_by: users.sample(1).first,
+            urgent_flag: flag,
+            line: lines[idx%3], # thank you seeds.rb! 
```

Comment:
this is a cool solution to picking a line - part of programming is having a good eye for what ideas to borrow and this is great